taipy/gui/gui.py

```diff
@@ -1002,7 +1002,9 @@ def __send_var_list_update(  # noqa C901
                 elif isinstance(newvalue, _TaipyToJson):
                     newvalue = newvalue.get()
                 if isinstance(newvalue, (dict, _MapDict)):
-                    continue  # this var has no transformer
+                    # Skip in taipy-gui, available in custom frontend
+                    if request.cookies.get(_Server._RESOURCE_HANDLER_ARG, None) is None:
```

Comment:
Are we sure the request is always available ?
what if the update is done from a long callback ?

Reply:
should we at least try/except this or check the availability or request and request.cookies ?

Reply:
I more than agree with you!